packages/nextjs/src/index.types.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/export */
+
+// We export everything from both the client part of the SDK and from the server part. Some of the exports collide,
+// which is not allowed, unless we redifine the colliding exports in this file - which we do below.
+export * from './config';
+export * from './client';
+export * from './server';
```

Comment:
This file is just for generating a `.d.ts`. We're not transpiling it or pointing to it via `module`, `main`, or `browser` in the package.json so it can't end up in a bundle.